mirror_repos

```diff
@@ -1,7 +1,21 @@
 #! /bin/bash
 
+set -ueo pipefail
+
+export AWS_REGION=eu-west-1
```

Comment:
Can we do this in `eu-west-2` to be consistent with other stuff in the tools account?

Reply:
Done